consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -908,12 +905,17 @@ impl<
             // Check nullification exists in gap
             if !self.is_nullified(cursor) {
                 debug!(
-                    view = cursor,
+                    view = %cursor,
                     "missing nullification during proposal verification"
                 );
                 return None;
             }
-            cursor -= 1;
+
+            if let Some(previous) = cursor.previous() {
```

Comment:
Can we `.expect()` here? The code before didn't have a problem assuming non-zero